cargo-frc/src/toolchain.rs

```diff
@@ -0,0 +1,160 @@
+#[cfg(target_os = "linux")]
+mod os {
+    use std::path::PathBuf;
+
+    pub fn install_dir() -> PathBuf {
+        dirs::home_dir()
+            .expect("Can't determine home directory")
+            .join("wpilib")
+        //.ok_or("Can't determine home directory".to_owned())
+    }
+
+    pub const TOOLCHAIN_URL_2019: &str = "https://github.com/wpilibsuite/toolchain-builder/releases/download/v2019-3/FRC-2019-Linux-Toolchain-6.3.0.tar.gz";
+    pub const TOOLCHAIN_URL_2020: &str = "https://github.com/wpilibsuite/roborio-toolchain/releases/download/v2020-2/FRC-2020-Linux-Toolchain-7.3.0.tar.gz";
+}
+
+#[cfg(target_os = "macos")]
+mod os {
+    use std::path::PathBuf;
+
+    pub fn install_dir() -> PathBuf {
+        dirs::home_dir()
+            .expect("Can't determine home directory")
+            .join("wpilib")
+        //.ok_or("Can't determine home directory".to_owned())
+    }
+
+    const TOOLCHAIN_URL_2019: &str = "https://github.com/wpilibsuite/toolchain-builder/releases/download/v2019-3/FRC-2019-Mac-Toolchain-6.3.0.tar.gz";
+    const TOOLCHAIN_URL_2020: &str = "https://github.com/wpilibsuite/roborio-toolchain/releases/download/v2020-2/FRC-2020-Mac-Toolchain-7.3.0.tar.gz";
```

Comment:
This gets very repetitive… wouldn't it be sensible to build the URL from its components?

Reply:
It might be. I went with just directly including the URLs because the toolchain download location has changed twice in the past two years. Building from components would make it easier to install specific versions, or let the user override the version if a newer version is released midseason without needing to update cargo-frc itself. We could potentially even query the github api to see what releases are available, but I'm not sure if that is necessary. I'll think about this some more.

Reply:
I don't expect the toolchain repo to change location again.